src-docs/src/views/selection_controls/guidelines.js

```diff
@@ -209,6 +307,110 @@ export default () => {
         </GuideRuleExample>
       </GuideRule>
 
+      <GuideRule
+        heading="Checkbox and radio button labels"
```

Comment:
Since we already exemplify checkbox/radio labels in the section above, I suggest renaming this to be more legend specific, and then probably linking to it from the [Fieldset & legend section](https://eui.elastic.co/pr_5707/#/forms/selection-controls#fieldset-and-legend) like we did for switches.

```suggestion
        heading="Legends"
```

Reply:
Good point thanks! I made a 'legend' section in [this commit](https://github.com/elastic/eui/pull/5707/commits/1da4dcd17aadcd467188ffb280b400997de79bf6) and added a link as you suggested.